<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsViewPreferences</class>
 <widget class="QWidget" name="ResultsViewPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Results display</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Style:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="oneLine">
          <property name="text">
           <string>One line</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="twoLines">
          <property name="text">
           <string>Two lines</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Number of results per page:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="nbResultsDefault">
            <property name="text">
             <string>Default</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="nbResults">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Results sorting:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="resultsOrder">
          <item>
           <property name="text">
            <string>By relevance</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>By score, then relevance</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fontsBox">
     <property name="title">
      <string>Fonts</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="previewBox">
     <property name="title">
      <string>Preview</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
